File: Homework/JavaScript-Fundamentals/08. Functions/english-digit.js
function solve(args) {
    console.log(lastDigitToWord(args[0]));

    function lastDigitToWord(number) {
        var digits =
            ["zero", "one", "two", "three",
                "four", "five", "six", "seven",
                "eight", "nine"];

        return digits[number.toString().split('').pop()];
    }
}

File: Homework/CSharp-Part-2/03. Methods/EnglishDigit.cs
using System;

class EnglishDigit
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());

        Console.WriteLine(LastDigitToWord(number));

    }

    static string LastDigitToWord(int number)
    {
        string[] digits = 
        {
            "zero", "one", "two", "three",
            "four", "five", "six", "seven",
            "eight", "nine"
        };

        return digits[number % 10]; 
    }
}